Januaryreleasefixing

Thinking-waffle opened the pull request.

Harain committed:
added tfv_units_cavalry.asset

moved 4 entities to r56_units_cavalry.asset
fix for:
[18:48:42][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity RAJ_cavalry_entity, in file "gfx/entities/r56_units_cavalry.asset", near line 1446
[18:48:42][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity RAJ_cavalry_2_entity, in file "gfx/entities/r56_units_cavalry.asset", near line 1447
[18:48:42][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity RAJ_cavalry_entity, in file "gfx/entities/r56_units_cavalry.asset", near line 1452
[18:48:42][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity RAJ_cavalry_2_entity, in file "gfx/entities/r56_units_cavalry.asset", near line 1453

Harain committed:
added version to entities

to prevent:
[19:54:50][no_game_date][pdx_entity.cpp:2171]: Duplicate of LAT_infantry_entity added to entity system

ItsFlor committed:
liberal lux fixes

Harain committed:
added WW_units_tanks.asset

and added versions to BEL_light_armor_entity to avoid this
[21:26:38][no_game_date][pdx_entity.cpp:2276]: Duplicate of BEL_light_armor_entity added to entity system

Harain committed:
added entity versions

another example of weird error logging:
the duplicate entity is actually the clone BEL_medium_armor_entity, but it reports the parent :P
[21:36:13][no_game_date][pdx_entity.cpp:2276]: Duplicate of FRA_medium_armor_entity added to entity system

Harain committed:
added tfv_units_tanks.asset

moved entities from vanilla to our file, so that parents and clone are in the same file, to avoid these errors:
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity CAN_light_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1383
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity CAN_medium_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1388
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity CAN_light_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1407
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity CAN_medium_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1411
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity CAN_light_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1430
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity AST_medium_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1435
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity AST_medium_armor_2_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1438
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity NZL_light_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1452
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity NZL_medium_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1454
[21:53:47][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity CAN_light_armor_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1602

Harain committed:
added version to entity

to stop this error, because there's already an ITA_medium_armor_entity in vanilla
[22:48:38][no_game_date][pdx_entity.cpp:2276]: Duplicate of ITA_medium_armor_entity added to entity system

Harain committed:
duplicate entity

those entities commented out aren't needed, accoding to thinking_waffle
[23:33:54][no_game_date][pdx_entity.cpp:2171]: Duplicate of PER_light_armor_0_entity added to entity system

Harain committed:
fixed EGY_secure_the_bosporus wargoal

was granting against Greece instead of Turkey

Harain committed:
updated PRU_fascism.tga

just opened and saved it in correct format (with Paint.NET)
[11:29:10][1936.06.22.07][flagtextureatlas.cpp:203]: Error loading country flag overlay: gfx/flags/small/PRU_fascism.tga : Unsupported format. Expected 32bpp, uncompressed, no color palette

Harain committed:
error log fix

massive cascade of errors of that mio not existing (it doesn't if you own Gotterdamerung)

Harain committed:
added/improved comments in asset files

Harain committed:
added version to duplicate entities

also reorganized file a bit
fix for:
[12:02:24][no_game_date][pdx_entity.cpp:2276]: Duplicate of ITA_heavy_armor_entity added to entity system
[12:02:24][no_game_date][pdx_entity.cpp:2276]: Duplicate of GER_light_armor_entity added to entity system
[12:42:13][no_game_date][pdx_entity.cpp:2276]: Duplicate of GER_medium_armor_entity added to entity system
[12:50:14][no_game_date][pdx_entity.cpp:2276]: Duplicate of GER_heavy_armor_entity added to entity system

Harain committed:
fix Niger entities typo

ItsFlor committed:
small lux election fix

Harain committed:
added version to duplicate entities

[13:28:14][no_game_date][pdx_entity.cpp:2276]: Duplicate of FRA_light_armor_entity added to entity system
[14:35:23][no_game_date][pdx_entity.cpp:2276]: Duplicate of ITA_light_armor_entity added to entity system

Harain committed:
commented unnecessary entities

no longer needed with tfv being integrated, to remove this error:
[14:48:48][no_game_date][pdx_entity.cpp:2171]: Duplicate of CAN_medium_armor_2_entity added to entity system

Harain committed:
commented duplicate vanilla entity

to remove this error:
[14:42:29][no_game_date][pdx_entity.cpp:2171]: Duplicate of NZL_super_heavy_armor_entity added to entity system

Harain committed:
moved CAN_medium_armor_2_entity

to avoid this error:
[15:35:59][no_game_date][assetfactory.cpp:876]: Couldn't find parent clone entity CAN_medium_armor_2_entity, in file "gfx/entities/r56_units_tanks.asset", near line 1498

Harain committed:
added version to entities

and commented another duplicate to prevent these errors:
[15:38:50][no_game_date][pdx_entity.cpp:2171]: Duplicate of BEL_artillery_entity added to entity system
[15:42:36][no_game_date][pdx_entity.cpp:2171]: Duplicate of LAT_artillery_entity added to entity system
[15:45:56][no_game_date][pdx_entity.cpp:2276]: Duplicate of anti_tank_entity added to entity system
[15:49:30][no_game_date][pdx_entity.cpp:2276]: Duplicate of artillery_entity added to entity system

ItsFlor committed:
another election fix and idea fix

Harain committed:
error log fix

SAU_encourage_general_creativity focus gives inventive_leadership_spirit, which isn't allowed without a Land Doctrine
causing the following error to occur daily until the country gets a Land Doctrine
[18:25:55][1937.11.12.01][politics.cpp:1484]: SAU couldn't remove/replace invalid idea inventive_leadership_spirit

Harain committed:
localization fix

for some reason that [BEL.GetAdjective] doesn't work in the tooltip, it appears exactly like that "[BEL.GetAdjective] Investments"
changing to plain ol' "Belgian"

Harain committed:
loading tip typo

Harain committed:
added version to entities

also fixed a typo in an entity: SAF instead of YUG
[10:46:50][no_game_date][pdx_entity.cpp:2276]: Duplicate of infantry_rifle_entity added to entity system
2x [11:19:52][no_game_date][pdx_entity.cpp:2276]: Duplicate of infantry_2_entity added to entity system
[11:49:42][no_game_date][pdx_entity.cpp:2276]: Duplicate of infantry_3_entity added to entity system

Harain committed:
last infantry duplicate

[12:03:32][no_game_date][pdx_entity.cpp:2276]: Duplicate of LAT_infantry_2_entity added to entity system

Harain committed:
updated units_planes.asset

Reordered much of the file to match vanilla (and added missing firebomb state),
but gets too messy halfway, will resume after my brain unmelts
fixed:
[13:06:24][no_game_date][pdx_entity.cpp:2171]: Duplicate of BRA_medium_plane_entity added to entity system
[13:38:25][no_game_date][pdx_entity.cpp:2171]: Duplicate of BRA_heavy_plane_entity added to entity system
[13:38:25][no_game_date][pdx_entity.cpp:2171]: Duplicate of BRA_fighter_equipment_2_entity added to entity system

Harain committed:
updated units_planes.asset

followup of previous commit
total rework of file, with vanilla on top (plus a few of our entities in the mix)
almost all R56 entities moved to the bottom
this makes it possible to use VSC compare, which allowd me to find several missing vanilla states (firebomb) and many entities
everything below ballistic_missile_equipment_1_entity is commented out for now, was all missing in R56

Harain committed:
fix Mongolian focus reward

added tech if country doesn't already have it, before adding trains

Harain committed:
error log fix

to prevent an error in cases where the capital doesn't exist yet (anarchists in spanish civil war for example)

Harain committed:
Mongolbank description typo

Harain committed:
minor event fix

added a check to actually add the template and prevent this error:
[23:07:45][1937.07.24.17][effectimplementation.cpp:3952]: common/decisions/ITA.txt:4255: add_units_to_division_template did not find template for Truppe Irregolari di Cammelli in add_units_to_division_template

Harain committed:
added version to several air entities

to clear the following errors:
[12:12:38][no_game_date][pdx_entity.cpp:2171]: Duplicate of CHL_light_plane_entity added to entity system
[12:12:38][no_game_date][pdx_entity.cpp:2171]: Duplicate of CHL_medium_plane_entity added to entity system
[12:12:38][no_game_date][pdx_entity.cpp:2171]: Duplicate of CHL_heavy_plane_entity added to entity system
[12:33:15][no_game_date][pdx_entity.cpp:2171]: Duplicate of CHL_jet_plane_entity added to entity system
[12:33:15][no_game_date][pdx_entity.cpp:2171]: Duplicate of ETH_light_plane_entity added to entity system
[12:33:15][no_game_date][pdx_entity.cpp:2171]: Duplicate of SWE_fighter_equipment_3_entity added to entity system
[12:33:15][no_game_date][pdx_entity.cpp:2171]: Duplicate of NOR_light_plane_entity added to entity system
[12:38:22][no_game_date][pdx_entity.cpp:2171]: Duplicate of AUS_fighter_equipment_0_entity added to entity system
[12:38:22][no_game_date][pdx_entity.cpp:2171]: Duplicate of AUS_medium_plane_entity added to entity system

Harain committed:
update units_planes.asset

deletd several duplicate entities and made some more file arrangements
fixes the following errors
[12:02:23][no_game_date][pdx_entity.cpp:2276]: Duplicate of fighter_equipment_0_entity added to entity system
[12:38:22][no_game_date][pdx_entity.cpp:2171]: Duplicate of GRE_light_plane_entity added to entity system

Harain committed:
air entities version changes

to clear 4 lines of the following error:
[13:03:40][no_game_date][pdx_entity.cpp:2276]: Duplicate of TUR_light_plane_entity added to entity system

Harain committed:
added version to air entities

to clear the following errors:
[13:11:54][no_game_date][pdx_entity.cpp:2171]: Duplicate of SWE_fighter_equipment_0_entity added to entity system
[13:11:54][no_game_date][pdx_entity.cpp:2171]: Duplicate of SWE_medium_plane_entity added to entity system
[13:11:54][no_game_date][pdx_entity.cpp:2171]: Duplicate of SWE_heavy_plane_entity added to entity system
[13:18:25][no_game_date][pdx_entity.cpp:2171]: Duplicate of NOR_heavy_plane_entity added to entity system
[13:18:25][no_game_date][pdx_entity.cpp:2171]: Duplicate of DEN_heavy_plane_entity added to entity system
[13:20:41][no_game_date][pdx_entity.cpp:2171]: Duplicate of MEX_heavy_plane_entity added to entity system
[13:20:41][no_game_date][pdx_entity.cpp:2171]: Duplicate of BRA_jet_plane_entity added to entity system
[13:20:41][no_game_date][pdx_entity.cpp:2171]: Duplicate of SWE_jet_plane_entity added to entity system

Harain committed:
removed duplicate entities

were repeated twice, causing the following error:
[13:20:41][no_game_date][pdx_entity.cpp:2171]: Duplicate of SPR_anarchist_spain_medium_plane_entity added to entity system

Harain committed:
added version to air entities

to clear 3 lines of the following error:
[13:30:21][no_game_date][pdx_entity.cpp:2276]: Duplicate of BYZ_UNIFIED_light_plane_entity added to entity system

ItsFlor committed:
spelling

Harain committed:
last air entity error

seems you can't have a version 2 without a version 1 or you get a false positive
[13:31:23][no_game_date][pdx_entity.cpp:2276]: Duplicate of fighter_equipment_0_entity added to entity system

Harain committed:
GFX fix

[14:36:51][1938.03.17.01][icon_entry.cpp:37]: Icon definition "r56_portrait_generic_operative_europe_5" does neither look like a file path nor a GFX entry as it does not start with "GFX_"

marijn211 reviewed (commented):
Looks good so far except the probably broken exists check I told Harain about, havent checked error log yet

Harain committed:
fix my mistake

Harain committed:
fix trigger in Will of the People

must use a custom trigger tooltip or the OVERLORD line will always display errors for independent countries, since it needed to be displayed in a tooltip
was getting dozens of this while mousing over it
[1938.09.01.02][trigger.cpp:456]: common/ideas/r56i_laws_leadership.txt:203: has_government: Invalid Scope, supported: Country, provided: None

Harain committed:
Polish focus reward fix

ship seems to be a CL but was getting a heavy gun, causing an error with rest of modules:
[1940.01.24.01][equipment_effects.cpp:808]: 'common/national_focus/poland.txt:15235: create_equipment_variant': 'D'Entrecasteaux Class' - Invalid module setup. Heavy Cruiser Battery I disallow modules of category Rapid Fire Guns

Harain committed:
rework units_vehicles.asset

made it all like vanilla up to line 2591, with only a couple of R56 changes
also added a few missing vanilla entities, commented for now until thinking_waffle checks them
moved all our stuff below that to make it easier to compare with vanilla

Thinking-waffle committed:
should fix bad scope of base game city fall events trigger

Harain committed:
moved duplicate bycicle entities from units_vehicles.asset to units_infantry.asset

changes in units_vehicles.assetup to line 2590 are only adjustments to match vanilla which it seems I forgot to save in last commit... sorry bout that *blush*

these were actual duplicates, not false positives or missing version
removes the following errors:
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of bicycle_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2276]: Duplicate of bicycle_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2276]: Duplicate of generic_bicycle_vehicle_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2276]: Duplicate of generic_bicycle_rifle_combined_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2276]: Duplicate of generic_bicycle_rider_rifle_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2276]: Duplicate of generic_bicycle_2_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2276]: Duplicate of generic_bicycle_mg_combined_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2276]: Duplicate of generic_bicycle_rider_mg_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of GER_bicycle_rifle_combined_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of GER_bicycle_rider_rifle_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of GER_bicycle_vehicle_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of GER_bicycle_2_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of GER_bicycle_mg_combined_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of GER_bicycle_rider_mg_entity added to entity system

Thinking-waffle committed:
Update 00_tank_icons.txt

Thinking-waffle committed:
Add files via upload

Thinking-waffle committed:
Add files via upload

Thinking-waffle committed:
Update _r56_leader_portraits.gfx

Thinking-waffle committed:
Update ENG.txt

Thinking-waffle committed:
Update USA.txt

Thinking-waffle committed:
Update 00_tank_icons.txt

Thinking-waffle committed:
Update 00_tank_icons.txt

Thinking-waffle committed:
Update 00_tank_icons.txt

Thinking-waffle committed:
Update PER.txt

Thinking-waffle committed:
Update japan.txt

Thinking-waffle committed:
Update uk.txt

Thinking-waffle committed:
Update Cuba.txt

Thinking-waffle committed:
added HUN_scripted_triggers

will be tweaked to harmonized conditions for Austro-Hungarian shared tree with the r56 tags.

Thinking-waffle committed:
Update CRO.txt

Thinking-waffle committed:
Update CRO - Croatia.txt

Thinking-waffle committed:
Update WUW_Austria.txt

Thinking-waffle committed:
Update HUN_scripted_triggers.txt

Thinking-waffle committed:
added shared Austria Hungary tree

Necessary because coordinates have to be adjusted to our trees.

Thinking-waffle committed:
Update habsburg_joint.txt

Thinking-waffle committed:
Update generic.txt

Thinking-waffle committed:
Update habsburg_joint.txt

Thinking-waffle committed:
Update albania.txt

Thinking-waffle committed:
Update croatia.txt

Harain committed:
fix r56 focus tooltip for Niels Bohr

Thinking-waffle committed:
Update slovakia.txt

Harain committed:
last duplicate vehicle entities fix

added version to fix the following errors:
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of BEL_motorized_entity added to entity system
[22:49:42][no_game_date][pdx_entity.cpp:2171]: Duplicate of BEL_bicycle_entity added to entity system
[23:38:11][no_game_date][pdx_entity.cpp:2171]: Duplicate of BEL_bicycle_rifle_combined_entity added to entity system
[23:38:11][no_game_date][pdx_entity.cpp:2171]: Duplicate of BEL_bicycle_rider_rifle_entity added to entity system
[23:38:11][no_game_date][pdx_entity.cpp:2171]: Duplicate of BEL_bicycle_2_entity added to entity system
[23:38:11][no_game_date][pdx_entity.cpp:2171]: Duplicate of BEL_bicycle_mg_combined_entity added to entity system
[23:38:11][no_game_date][pdx_entity.cpp:2171]: Duplicate of BEL_bicycle_rider_mg_entity added to entity system
[23:38:11][no_game_date][pdx_entity.cpp:2171]: Duplicate of LAT_motorized_entity added to entity system
[23:38:11][no_game_date][pdx_entity.cpp:2171]: Duplicate of LAT_mechanized_entity added to entity system
[23:38:11][no_game_date][pdx_entity.cpp:2171]: Duplicate of LAT_vehicle_infantry_rifle_entity added to entity system

marijn211 committed:
Fix broken loc for  GER gamerule

BiccyThiccy committed:
LIT Focus - Relative Position Error fix

SpicyAlfredo committed:
exile unit fix

SpicyAlfredo committed:
reverting an very old Mas Mob buff

SpicyAlfredo committed:
Small Mass Assaults Adjustments

Harain committed:
redundant word in event text

Harain committed:
Egypt division numbers fix

Thinking-waffle committed:
Update ENG - Britain.txt

Thinking-waffle committed:
Update hungary_wuw.txt

Harain committed:
fix heavy tank land project

fix for:
[11:45:57][no_game_date][common/special_projects/projects/r56_land_projects.txt:535]: Value does not belong to designated Script EnumType: heavy_armor not in script_enum_equipment_bonus_type

Harain committed:
added 03_plane_icons.txt

replaced vanilla supersonic_fighter_equipment_1 with r56 jet_fighter_equipment_x
fix for 8 of these errors:
8x [11:45:58][no_game_date][equipment_graphic_database.cpp:93]: Unknown equipment type: supersonic_fighter_equipment_1
